cypress/integration/advanced-functionalities/admin/system/integration-gateway/Create.spec.js

```diff
@@ -0,0 +1,62 @@
+/// <reference types="cypress" />
+const adminURL = Cypress.env('ADMIN_URL')
+const email = Cypress.env('USER_EMAIL')
+const password = Cypress.env('USER_PASSWORD')
+
+describe('Test for creating an integration gateway', () => {
+  before(() => {
+    if (!window.sessionStorage.getItem('auth.refresh-token')) {
+      cy.login({ email, password, url: adminURL })
+    }
+  })
+
+  context('Test for creating an integration gateway', () => {
+    it('should check whether the correct buttons and states are present', () => {
+      cy.visit(adminURL + '/')
+      // We wait 3s in order the page to be fully loaded
+      cy.wait(3000)
+      cy.get('.nav-sidebar').contains('Integration Gateway').click()
+      // We wait 2s in order the page to be fully loaded
+      cy.wait(2000)
+      cy.get('[data-test-id="button-add"]').click()
+      // We wait 1s in order the page to be fully loaded
+      cy.wait(1000)
+      cy.get('[data-test-id="button-delete"]').should('not.exist')
+      cy.get('[data-test-id="button-undelete"]').should('not.exist')
+      cy.get('[data-test-id="button-submit"].disabled').should('exist')
+      cy.get('[data-test-id="button-add"]').should('not.exist')
+      cy.get('[data-test-id="input-created-at"]').should('not.exist')
+      cy.get('[data-test-id="input-updated-at"]').should('not.exist')
+      cy.get('[data-test-id="input-deleted-at"]').should('not.exist')
+      cy.get('[data-test-id="select-method"]').select('POST')
+      cy.get('[data-test-id="select-method"]').should('have.value', 'POST')
+      cy.get('[data-test-id="select-method"]').select('PUT')
+      cy.get('[data-test-id="select-method"]').should('have.value', 'PUT')
+      cy.get('[data-test-id="select-method"]').select('DELETE')
+      cy.get('[data-test-id="select-method"]').should('have.value', 'DELETE')
+      cy.get('[data-test-id="select-method"]').select('GET')
+      cy.get('[data-test-id="select-method"]').should('have.value', 'GET')
+    })
+
+    it('should not be able to create an integration gateway with a misconfigured field', () => {
+      cy.get('[data-test-id="input-endpoint"]').type('name')
+      // We checked if the field is red/invalid
+      cy.get('[data-test-id="input-endpoint"].is-invalid').should('exist')
```

Comment:
One more thing: every endpoint must start with `/` => `/endpoint`. After adding `/` to the endpoint field, check if the warning text below the field has changed